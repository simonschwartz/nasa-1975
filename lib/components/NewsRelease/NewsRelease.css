.NASA1975-news-release-header {
  border-bottom: 3px solid #171717;
  margin-bottom: 1.25em;
  padding-bottom: 0.25em;
}
