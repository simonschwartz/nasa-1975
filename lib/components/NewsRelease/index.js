import NewsRelease from "./NewsRelease";

export default NewsRelease;
