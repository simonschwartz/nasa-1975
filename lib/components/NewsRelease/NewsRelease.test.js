import React from "react";
import ReactDOM from "react-dom";
import renderer from "react-test-renderer";
import NewsRelease from "./";

it("renders correct markup with no props", () => {
  const tree = renderer
    .create(
      <NewsRelease authorName="Neil Armstrong" title="Test title">
        <p>Test article content</p>
      </NewsRelease>
    )
    .toJSON();
  expect(tree).toMatchSnapshot();
});

it("renders correct markup with authorLocation prop", () => {
  const tree = renderer
    .create(
      <NewsRelease
        authorName="Neil Armstrong"
        title="Test title"
        authorLocation="Houston, Texas"
      >
        <p>Test article content</p>
      </NewsRelease>
    )
    .toJSON();
  expect(tree).toMatchSnapshot();
});

it("renders correct markup with authorPhone prop", () => {
  const tree = renderer
    .create(
      <NewsRelease
        authorName="Neil Armstrong"
        title="Test title"
        authorPhone="0752/1234-087"
      >
        <p>Test article content</p>
      </NewsRelease>
    )
    .toJSON();
  expect(tree).toMatchSnapshot();
});

it("renders correct markup with centerName prop", () => {
  const tree = renderer
    .create(
      <NewsRelease
        authorName="Neil Armstrong"
        title="Test title"
        centerName="Mission Control"
      >
        <p>Test article content</p>
      </NewsRelease>
    )
    .toJSON();
  expect(tree).toMatchSnapshot();
});

it("renders correct markup with centerAddress prop", () => {
  const tree = renderer
    .create(
      <NewsRelease
        authorName="Neil Armstrong"
        title="Test title"
        centerAddress={["Washington. D.C. 20546 ", "AC 202 755-8370"]}
      >
        <p>Test article content</p>
      </NewsRelease>
    )
    .toJSON();
  expect(tree).toMatchSnapshot();
});

it("renders correct markup with multiple props", () => {
  const tree = renderer
    .create(
      <NewsRelease
        title="Skylab puts out welcome mat for comet"
        authorName="John Smith"
        authorLocation="Goddard Space Flight Center"
        authorPhone="301/725-1234"
        centerName="Goddard space center"
        centerAddress={["Washington. D.C. 20546 ", "AC 202 755-8370"]}
        style={{
          maxWidth: "700px",
          margin: "auto"
        }}
      >
        <p>Test article content</p>
      </NewsRelease>
    )
    .toJSON();
  expect(tree).toMatchSnapshot();
});

it("renders without crashing", () => {
  const div = document.createElement("div");
  ReactDOM.render(
    <NewsRelease authorName="Neil Armstrong" title="Test title">
      <p>Test article content</p>
    </NewsRelease>,
    div
  );
});
