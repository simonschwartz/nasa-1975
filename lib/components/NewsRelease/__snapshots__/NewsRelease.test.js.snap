// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`renders correct markup with authorLocation prop 1`] = `
<article
  className="NASA1975-news-release "
>
  <header>
    <section
      className="NASA1975-stem-logo NASA1975-stem-logo--lg NASA1975-stem-logo--red "
    >
      <svg
        aria-labelledby="title desc"
        className="NASA1975-stem-logo-container"
        role="img"
        viewBox="0 0 508.204 141.732"
        xmlns="http://www.w3.org/2000/svg"
      >
        <title
          id="title"
        >
          NASA logo
        </title>
        <desc
          id="desc"
        >
          NASA Logo
        </desc>
        <g>
          <path
            d="M91.991,104.699c1.576,5.961,4.119,8.266,8.613,8.266c4.659,0,7.102-2.799,7.102-8.266V3.2h29.184v101.499    c0,14.307-1.856,20.506-9.11,27.762c-5.228,5.229-14.871,9.271-27.047,9.271c-9.837,0-19.25-3.256-25.253-9.27    c-5.263-5.273-8.154-10.689-12.672-27.764L44.9,37.033c-1.577-5.961-4.119-8.265-8.613-8.265c-4.66,0-7.103,2.798-7.103,8.265    v101.5H0v-101.5C0,22.727,1.857,16.527,9.111,9.271C14.337,4.044,23.981,0,36.158,0c9.837,0,19.25,3.257,25.253,9.27    c5.263,5.273,8.154,10.689,12.672,27.764L91.991,104.699z"
            fill="#e30613"
          />
          <path
            d="M478.038,138.533L444.334,33.096c-0.372-1.164-0.723-2.152-1.263-2.811    c-0.926-1.127-2.207-1.719-3.931-1.719c-1.723,0-3.004,0.592-3.931,1.719c-0.539,0.658-0.891,1.646-1.262,2.811l-33.703,105.437    h-30.167l36.815-115.177c1.918-6,4.66-11.094,8.139-14.488C421.002,3.047,428.038,0,439.141,0s18.14,3.047,24.109,8.867    c3.479,3.395,6.221,8.488,8.14,14.488l36.814,115.177H478.038z"
            fill="#e30613"
          />
          <path
            d="M328.878,138.533c19.12,0,28.446-4.062,35.814-11.389c8.153-8.105,12.053-16.973,12.053-30.213    c0-11.699-4.283-22.535-10.804-29.019c-8.526-8.479-19.116-11.151-36.384-11.151L305.37,56.76c-9.242,0-12.925-1.117-15.839-3.98    c-2.001-1.964-2.939-4.885-2.939-8.328c0-3.559,0.857-7.074,3.303-9.475c2.171-2.131,5.13-3.109,10.816-3.109h69.903V3.2H306.05    c-19.12,0-28.445,4.063-35.814,11.389c-8.152,8.105-12.053,16.972-12.053,30.212c0,11.701,4.283,22.536,10.804,29.019    c8.527,8.479,19.116,11.152,36.384,11.152l24.188,0.002c9.242,0,12.925,1.115,15.839,3.979c2.001,1.965,2.939,4.885,2.939,8.328    c0,3.559-0.857,7.074-3.302,9.475c-2.172,2.131-5.131,3.109-10.817,3.109h-72.094l-27.651-86.509    c-1.918-6-4.66-11.094-8.139-14.488C220.363,3.047,213.327,0,202.224,0s-18.14,3.047-24.108,8.867    c-3.48,3.395-6.221,8.488-8.139,14.488l-36.815,115.177h30.166l33.704-105.437c0.372-1.164,0.723-2.152,1.263-2.811    c0.926-1.127,2.208-1.719,3.931-1.719s3.004,0.592,3.931,1.719c0.54,0.658,0.891,1.646,1.262,2.811l33.704,105.437H328.878z"
            fill="#e30613"
          />
        </g>
      </svg>
      <h1
        className="NASA1975-heading NASA1975-heading--lg NASA1975-heading--light NASA1975-stem-logo-heading"
      >
        News
      </h1>
    </section>
    <div
      className="NASA1975-news-release-header"
    >
      <h2
        className="NASA1975-heading NASA1975-heading--xs NASA1975-heading--light NASA1975-agency-name NASA1975-agency-name--inline"
      >
        <span
          className="NASA1975-agency-name-N"
        >
          National
        </span>
         
        <span
          className="NASA1975-agency-name-Aa"
        >
          Aeronautics and
        </span>
         
        <span
          className="NASA1975-agency-name-SA"
        >
          <span
            className="NASA1975-agency-name-S"
          >
            Space
          </span>
           
          <span
            className="NASA1975-agency-name-A"
          >
            Administration
          </span>
        </span>
      </h2>
      
      <h2
        className="NASA1975-heading NASA1975-heading--xs NASA1975-heading--light NASA1975-agency-name-center-address"
      />
    </div>
    <ul
      className="NASA1975-article-author "
    >
      <li>
        Neil Armstrong
      </li>
      <li>
        Houston, Texas
      </li>
      
    </ul>
    <h1
      className="NASA1975-heading NASA1975-heading--lg NASA1975-heading--medium "
    >
      Test title
    </h1>
  </header>
  <div
    className="NASA1975"
  >
    <p>
      Test article content
    </p>
  </div>
</article>
`;

exports[`renders correct markup with authorPhone prop 1`] = `
<article
  className="NASA1975-news-release "
>
  <header>
    <section
      className="NASA1975-stem-logo NASA1975-stem-logo--lg NASA1975-stem-logo--red "
    >
      <svg
        aria-labelledby="title desc"
        className="NASA1975-stem-logo-container"
        role="img"
        viewBox="0 0 508.204 141.732"
        xmlns="http://www.w3.org/2000/svg"
      >
        <title
          id="title"
        >
          NASA logo
        </title>
        <desc
          id="desc"
        >
          NASA Logo
        </desc>
        <g>
          <path
            d="M91.991,104.699c1.576,5.961,4.119,8.266,8.613,8.266c4.659,0,7.102-2.799,7.102-8.266V3.2h29.184v101.499    c0,14.307-1.856,20.506-9.11,27.762c-5.228,5.229-14.871,9.271-27.047,9.271c-9.837,0-19.25-3.256-25.253-9.27    c-5.263-5.273-8.154-10.689-12.672-27.764L44.9,37.033c-1.577-5.961-4.119-8.265-8.613-8.265c-4.66,0-7.103,2.798-7.103,8.265    v101.5H0v-101.5C0,22.727,1.857,16.527,9.111,9.271C14.337,4.044,23.981,0,36.158,0c9.837,0,19.25,3.257,25.253,9.27    c5.263,5.273,8.154,10.689,12.672,27.764L91.991,104.699z"
            fill="#e30613"
          />
          <path
            d="M478.038,138.533L444.334,33.096c-0.372-1.164-0.723-2.152-1.263-2.811    c-0.926-1.127-2.207-1.719-3.931-1.719c-1.723,0-3.004,0.592-3.931,1.719c-0.539,0.658-0.891,1.646-1.262,2.811l-33.703,105.437    h-30.167l36.815-115.177c1.918-6,4.66-11.094,8.139-14.488C421.002,3.047,428.038,0,439.141,0s18.14,3.047,24.109,8.867    c3.479,3.395,6.221,8.488,8.14,14.488l36.814,115.177H478.038z"
            fill="#e30613"
          />
          <path
            d="M328.878,138.533c19.12,0,28.446-4.062,35.814-11.389c8.153-8.105,12.053-16.973,12.053-30.213    c0-11.699-4.283-22.535-10.804-29.019c-8.526-8.479-19.116-11.151-36.384-11.151L305.37,56.76c-9.242,0-12.925-1.117-15.839-3.98    c-2.001-1.964-2.939-4.885-2.939-8.328c0-3.559,0.857-7.074,3.303-9.475c2.171-2.131,5.13-3.109,10.816-3.109h69.903V3.2H306.05    c-19.12,0-28.445,4.063-35.814,11.389c-8.152,8.105-12.053,16.972-12.053,30.212c0,11.701,4.283,22.536,10.804,29.019    c8.527,8.479,19.116,11.152,36.384,11.152l24.188,0.002c9.242,0,12.925,1.115,15.839,3.979c2.001,1.965,2.939,4.885,2.939,8.328    c0,3.559-0.857,7.074-3.302,9.475c-2.172,2.131-5.131,3.109-10.817,3.109h-72.094l-27.651-86.509    c-1.918-6-4.66-11.094-8.139-14.488C220.363,3.047,213.327,0,202.224,0s-18.14,3.047-24.108,8.867    c-3.48,3.395-6.221,8.488-8.139,14.488l-36.815,115.177h30.166l33.704-105.437c0.372-1.164,0.723-2.152,1.263-2.811    c0.926-1.127,2.208-1.719,3.931-1.719s3.004,0.592,3.931,1.719c0.54,0.658,0.891,1.646,1.262,2.811l33.704,105.437H328.878z"
            fill="#e30613"
          />
        </g>
      </svg>
      <h1
        className="NASA1975-heading NASA1975-heading--lg NASA1975-heading--light NASA1975-stem-logo-heading"
      >
        News
      </h1>
    </section>
    <div
      className="NASA1975-news-release-header"
    >
      <h2
        className="NASA1975-heading NASA1975-heading--xs NASA1975-heading--light NASA1975-agency-name NASA1975-agency-name--inline"
      >
        <span
          className="NASA1975-agency-name-N"
        >
          National
        </span>
         
        <span
          className="NASA1975-agency-name-Aa"
        >
          Aeronautics and
        </span>
         
        <span
          className="NASA1975-agency-name-SA"
        >
          <span
            className="NASA1975-agency-name-S"
          >
            Space
          </span>
           
          <span
            className="NASA1975-agency-name-A"
          >
            Administration
          </span>
        </span>
      </h2>
      
      <h2
        className="NASA1975-heading NASA1975-heading--xs NASA1975-heading--light NASA1975-agency-name-center-address"
      />
    </div>
    <ul
      className="NASA1975-article-author "
    >
      <li>
        Neil Armstrong
      </li>
      
      <li>
        (Phone: 
        0752/1234-087
        )
      </li>
    </ul>
    <h1
      className="NASA1975-heading NASA1975-heading--lg NASA1975-heading--medium "
    >
      Test title
    </h1>
  </header>
  <div
    className="NASA1975"
  >
    <p>
      Test article content
    </p>
  </div>
</article>
`;

exports[`renders correct markup with centerAddress prop 1`] = `
<article
  className="NASA1975-news-release "
>
  <header>
    <section
      className="NASA1975-stem-logo NASA1975-stem-logo--lg NASA1975-stem-logo--red "
    >
      <svg
        aria-labelledby="title desc"
        className="NASA1975-stem-logo-container"
        role="img"
        viewBox="0 0 508.204 141.732"
        xmlns="http://www.w3.org/2000/svg"
      >
        <title
          id="title"
        >
          NASA logo
        </title>
        <desc
          id="desc"
        >
          NASA Logo
        </desc>
        <g>
          <path
            d="M91.991,104.699c1.576,5.961,4.119,8.266,8.613,8.266c4.659,0,7.102-2.799,7.102-8.266V3.2h29.184v101.499    c0,14.307-1.856,20.506-9.11,27.762c-5.228,5.229-14.871,9.271-27.047,9.271c-9.837,0-19.25-3.256-25.253-9.27    c-5.263-5.273-8.154-10.689-12.672-27.764L44.9,37.033c-1.577-5.961-4.119-8.265-8.613-8.265c-4.66,0-7.103,2.798-7.103,8.265    v101.5H0v-101.5C0,22.727,1.857,16.527,9.111,9.271C14.337,4.044,23.981,0,36.158,0c9.837,0,19.25,3.257,25.253,9.27    c5.263,5.273,8.154,10.689,12.672,27.764L91.991,104.699z"
            fill="#e30613"
          />
          <path
            d="M478.038,138.533L444.334,33.096c-0.372-1.164-0.723-2.152-1.263-2.811    c-0.926-1.127-2.207-1.719-3.931-1.719c-1.723,0-3.004,0.592-3.931,1.719c-0.539,0.658-0.891,1.646-1.262,2.811l-33.703,105.437    h-30.167l36.815-115.177c1.918-6,4.66-11.094,8.139-14.488C421.002,3.047,428.038,0,439.141,0s18.14,3.047,24.109,8.867    c3.479,3.395,6.221,8.488,8.14,14.488l36.814,115.177H478.038z"
            fill="#e30613"
          />
          <path
            d="M328.878,138.533c19.12,0,28.446-4.062,35.814-11.389c8.153-8.105,12.053-16.973,12.053-30.213    c0-11.699-4.283-22.535-10.804-29.019c-8.526-8.479-19.116-11.151-36.384-11.151L305.37,56.76c-9.242,0-12.925-1.117-15.839-3.98    c-2.001-1.964-2.939-4.885-2.939-8.328c0-3.559,0.857-7.074,3.303-9.475c2.171-2.131,5.13-3.109,10.816-3.109h69.903V3.2H306.05    c-19.12,0-28.445,4.063-35.814,11.389c-8.152,8.105-12.053,16.972-12.053,30.212c0,11.701,4.283,22.536,10.804,29.019    c8.527,8.479,19.116,11.152,36.384,11.152l24.188,0.002c9.242,0,12.925,1.115,15.839,3.979c2.001,1.965,2.939,4.885,2.939,8.328    c0,3.559-0.857,7.074-3.302,9.475c-2.172,2.131-5.131,3.109-10.817,3.109h-72.094l-27.651-86.509    c-1.918-6-4.66-11.094-8.139-14.488C220.363,3.047,213.327,0,202.224,0s-18.14,3.047-24.108,8.867    c-3.48,3.395-6.221,8.488-8.139,14.488l-36.815,115.177h30.166l33.704-105.437c0.372-1.164,0.723-2.152,1.263-2.811    c0.926-1.127,2.208-1.719,3.931-1.719s3.004,0.592,3.931,1.719c0.54,0.658,0.891,1.646,1.262,2.811l33.704,105.437H328.878z"
            fill="#e30613"
          />
        </g>
      </svg>
      <h1
        className="NASA1975-heading NASA1975-heading--lg NASA1975-heading--light NASA1975-stem-logo-heading"
      >
        News
      </h1>
    </section>
    <div
      className="NASA1975-news-release-header"
    >
      <h2
        className="NASA1975-heading NASA1975-heading--xs NASA1975-heading--light NASA1975-agency-name NASA1975-agency-name--inline"
      >
        <span
          className="NASA1975-agency-name-N"
        >
          National
        </span>
         
        <span
          className="NASA1975-agency-name-Aa"
        >
          Aeronautics and
        </span>
         
        <span
          className="NASA1975-agency-name-SA"
        >
          <span
            className="NASA1975-agency-name-S"
          >
            Space
          </span>
           
          <span
            className="NASA1975-agency-name-A"
          >
            Administration
          </span>
        </span>
      </h2>
      
      <h2
        className="NASA1975-heading NASA1975-heading--xs NASA1975-heading--light NASA1975-agency-name-center-address"
      >
        <span>
          Washington. D.C. 20546 
          <br />
        </span>
        <span>
          AC 202 755-8370
          <br />
        </span>
      </h2>
    </div>
    <ul
      className="NASA1975-article-author "
    >
      <li>
        Neil Armstrong
      </li>
      
      
    </ul>
    <h1
      className="NASA1975-heading NASA1975-heading--lg NASA1975-heading--medium "
    >
      Test title
    </h1>
  </header>
  <div
    className="NASA1975"
  >
    <p>
      Test article content
    </p>
  </div>
</article>
`;

exports[`renders correct markup with centerName prop 1`] = `
<article
  className="NASA1975-news-release "
>
  <header>
    <section
      className="NASA1975-stem-logo NASA1975-stem-logo--lg NASA1975-stem-logo--red "
    >
      <svg
        aria-labelledby="title desc"
        className="NASA1975-stem-logo-container"
        role="img"
        viewBox="0 0 508.204 141.732"
        xmlns="http://www.w3.org/2000/svg"
      >
        <title
          id="title"
        >
          NASA logo
        </title>
        <desc
          id="desc"
        >
          NASA Logo
        </desc>
        <g>
          <path
            d="M91.991,104.699c1.576,5.961,4.119,8.266,8.613,8.266c4.659,0,7.102-2.799,7.102-8.266V3.2h29.184v101.499    c0,14.307-1.856,20.506-9.11,27.762c-5.228,5.229-14.871,9.271-27.047,9.271c-9.837,0-19.25-3.256-25.253-9.27    c-5.263-5.273-8.154-10.689-12.672-27.764L44.9,37.033c-1.577-5.961-4.119-8.265-8.613-8.265c-4.66,0-7.103,2.798-7.103,8.265    v101.5H0v-101.5C0,22.727,1.857,16.527,9.111,9.271C14.337,4.044,23.981,0,36.158,0c9.837,0,19.25,3.257,25.253,9.27    c5.263,5.273,8.154,10.689,12.672,27.764L91.991,104.699z"
            fill="#e30613"
          />
          <path
            d="M478.038,138.533L444.334,33.096c-0.372-1.164-0.723-2.152-1.263-2.811    c-0.926-1.127-2.207-1.719-3.931-1.719c-1.723,0-3.004,0.592-3.931,1.719c-0.539,0.658-0.891,1.646-1.262,2.811l-33.703,105.437    h-30.167l36.815-115.177c1.918-6,4.66-11.094,8.139-14.488C421.002,3.047,428.038,0,439.141,0s18.14,3.047,24.109,8.867    c3.479,3.395,6.221,8.488,8.14,14.488l36.814,115.177H478.038z"
            fill="#e30613"
          />
          <path
            d="M328.878,138.533c19.12,0,28.446-4.062,35.814-11.389c8.153-8.105,12.053-16.973,12.053-30.213    c0-11.699-4.283-22.535-10.804-29.019c-8.526-8.479-19.116-11.151-36.384-11.151L305.37,56.76c-9.242,0-12.925-1.117-15.839-3.98    c-2.001-1.964-2.939-4.885-2.939-8.328c0-3.559,0.857-7.074,3.303-9.475c2.171-2.131,5.13-3.109,10.816-3.109h69.903V3.2H306.05    c-19.12,0-28.445,4.063-35.814,11.389c-8.152,8.105-12.053,16.972-12.053,30.212c0,11.701,4.283,22.536,10.804,29.019    c8.527,8.479,19.116,11.152,36.384,11.152l24.188,0.002c9.242,0,12.925,1.115,15.839,3.979c2.001,1.965,2.939,4.885,2.939,8.328    c0,3.559-0.857,7.074-3.302,9.475c-2.172,2.131-5.131,3.109-10.817,3.109h-72.094l-27.651-86.509    c-1.918-6-4.66-11.094-8.139-14.488C220.363,3.047,213.327,0,202.224,0s-18.14,3.047-24.108,8.867    c-3.48,3.395-6.221,8.488-8.139,14.488l-36.815,115.177h30.166l33.704-105.437c0.372-1.164,0.723-2.152,1.263-2.811    c0.926-1.127,2.208-1.719,3.931-1.719s3.004,0.592,3.931,1.719c0.54,0.658,0.891,1.646,1.262,2.811l33.704,105.437H328.878z"
            fill="#e30613"
          />
        </g>
      </svg>
      <h1
        className="NASA1975-heading NASA1975-heading--lg NASA1975-heading--light NASA1975-stem-logo-heading"
      >
        News
      </h1>
    </section>
    <div
      className="NASA1975-news-release-header"
    >
      <h2
        className="NASA1975-heading NASA1975-heading--xs NASA1975-heading--light NASA1975-agency-name NASA1975-agency-name--inline"
      >
        <span
          className="NASA1975-agency-name-N"
        >
          National
        </span>
         
        <span
          className="NASA1975-agency-name-Aa"
        >
          Aeronautics and
        </span>
         
        <span
          className="NASA1975-agency-name-SA"
        >
          <span
            className="NASA1975-agency-name-S"
          >
            Space
          </span>
           
          <span
            className="NASA1975-agency-name-A"
          >
            Administration
          </span>
        </span>
      </h2>
      <h2
        className="NASA1975-heading NASA1975-heading--xs NASA1975-heading--medium NASA1975-agency-name-center-name"
      >
        Mission Control
      </h2>
      <h2
        className="NASA1975-heading NASA1975-heading--xs NASA1975-heading--light NASA1975-agency-name-center-address"
      />
    </div>
    <ul
      className="NASA1975-article-author "
    >
      <li>
        Neil Armstrong
      </li>
      
      
    </ul>
    <h1
      className="NASA1975-heading NASA1975-heading--lg NASA1975-heading--medium "
    >
      Test title
    </h1>
  </header>
  <div
    className="NASA1975"
  >
    <p>
      Test article content
    </p>
  </div>
</article>
`;

exports[`renders correct markup with multiple props 1`] = `
<article
  className="NASA1975-news-release "
  style={
    Object {
      "margin": "auto",
      "maxWidth": "700px",
    }
  }
>
  <header>
    <section
      className="NASA1975-stem-logo NASA1975-stem-logo--lg NASA1975-stem-logo--red "
    >
      <svg
        aria-labelledby="title desc"
        className="NASA1975-stem-logo-container"
        role="img"
        viewBox="0 0 508.204 141.732"
        xmlns="http://www.w3.org/2000/svg"
      >
        <title
          id="title"
        >
          NASA logo
        </title>
        <desc
          id="desc"
        >
          NASA Logo
        </desc>
        <g>
          <path
            d="M91.991,104.699c1.576,5.961,4.119,8.266,8.613,8.266c4.659,0,7.102-2.799,7.102-8.266V3.2h29.184v101.499    c0,14.307-1.856,20.506-9.11,27.762c-5.228,5.229-14.871,9.271-27.047,9.271c-9.837,0-19.25-3.256-25.253-9.27    c-5.263-5.273-8.154-10.689-12.672-27.764L44.9,37.033c-1.577-5.961-4.119-8.265-8.613-8.265c-4.66,0-7.103,2.798-7.103,8.265    v101.5H0v-101.5C0,22.727,1.857,16.527,9.111,9.271C14.337,4.044,23.981,0,36.158,0c9.837,0,19.25,3.257,25.253,9.27    c5.263,5.273,8.154,10.689,12.672,27.764L91.991,104.699z"
            fill="#e30613"
          />
          <path
            d="M478.038,138.533L444.334,33.096c-0.372-1.164-0.723-2.152-1.263-2.811    c-0.926-1.127-2.207-1.719-3.931-1.719c-1.723,0-3.004,0.592-3.931,1.719c-0.539,0.658-0.891,1.646-1.262,2.811l-33.703,105.437    h-30.167l36.815-115.177c1.918-6,4.66-11.094,8.139-14.488C421.002,3.047,428.038,0,439.141,0s18.14,3.047,24.109,8.867    c3.479,3.395,6.221,8.488,8.14,14.488l36.814,115.177H478.038z"
            fill="#e30613"
          />
          <path
            d="M328.878,138.533c19.12,0,28.446-4.062,35.814-11.389c8.153-8.105,12.053-16.973,12.053-30.213    c0-11.699-4.283-22.535-10.804-29.019c-8.526-8.479-19.116-11.151-36.384-11.151L305.37,56.76c-9.242,0-12.925-1.117-15.839-3.98    c-2.001-1.964-2.939-4.885-2.939-8.328c0-3.559,0.857-7.074,3.303-9.475c2.171-2.131,5.13-3.109,10.816-3.109h69.903V3.2H306.05    c-19.12,0-28.445,4.063-35.814,11.389c-8.152,8.105-12.053,16.972-12.053,30.212c0,11.701,4.283,22.536,10.804,29.019    c8.527,8.479,19.116,11.152,36.384,11.152l24.188,0.002c9.242,0,12.925,1.115,15.839,3.979c2.001,1.965,2.939,4.885,2.939,8.328    c0,3.559-0.857,7.074-3.302,9.475c-2.172,2.131-5.131,3.109-10.817,3.109h-72.094l-27.651-86.509    c-1.918-6-4.66-11.094-8.139-14.488C220.363,3.047,213.327,0,202.224,0s-18.14,3.047-24.108,8.867    c-3.48,3.395-6.221,8.488-8.139,14.488l-36.815,115.177h30.166l33.704-105.437c0.372-1.164,0.723-2.152,1.263-2.811    c0.926-1.127,2.208-1.719,3.931-1.719s3.004,0.592,3.931,1.719c0.54,0.658,0.891,1.646,1.262,2.811l33.704,105.437H328.878z"
            fill="#e30613"
          />
        </g>
      </svg>
      <h1
        className="NASA1975-heading NASA1975-heading--lg NASA1975-heading--light NASA1975-stem-logo-heading"
      >
        News
      </h1>
    </section>
    <div
      className="NASA1975-news-release-header"
    >
      <h2
        className="NASA1975-heading NASA1975-heading--xs NASA1975-heading--light NASA1975-agency-name NASA1975-agency-name--inline"
      >
        <span
          className="NASA1975-agency-name-N"
        >
          National
        </span>
         
        <span
          className="NASA1975-agency-name-Aa"
        >
          Aeronautics and
        </span>
         
        <span
          className="NASA1975-agency-name-SA"
        >
          <span
            className="NASA1975-agency-name-S"
          >
            Space
          </span>
           
          <span
            className="NASA1975-agency-name-A"
          >
            Administration
          </span>
        </span>
      </h2>
      <h2
        className="NASA1975-heading NASA1975-heading--xs NASA1975-heading--medium NASA1975-agency-name-center-name"
      >
        Goddard space center
      </h2>
      <h2
        className="NASA1975-heading NASA1975-heading--xs NASA1975-heading--light NASA1975-agency-name-center-address"
      >
        <span>
          Washington. D.C. 20546 
          <br />
        </span>
        <span>
          AC 202 755-8370
          <br />
        </span>
      </h2>
    </div>
    <ul
      className="NASA1975-article-author "
    >
      <li>
        John Smith
      </li>
      <li>
        Goddard Space Flight Center
      </li>
      <li>
        (Phone: 
        301/725-1234
        )
      </li>
    </ul>
    <h1
      className="NASA1975-heading NASA1975-heading--lg NASA1975-heading--medium "
    >
      Skylab puts out welcome mat for comet
    </h1>
  </header>
  <div
    className="NASA1975"
  >
    <p>
      Test article content
    </p>
  </div>
</article>
`;

exports[`renders correct markup with no props 1`] = `
<article
  className="NASA1975-news-release "
>
  <header>
    <section
      className="NASA1975-stem-logo NASA1975-stem-logo--lg NASA1975-stem-logo--red "
    >
      <svg
        aria-labelledby="title desc"
        className="NASA1975-stem-logo-container"
        role="img"
        viewBox="0 0 508.204 141.732"
        xmlns="http://www.w3.org/2000/svg"
      >
        <title
          id="title"
        >
          NASA logo
        </title>
        <desc
          id="desc"
        >
          NASA Logo
        </desc>
        <g>
          <path
            d="M91.991,104.699c1.576,5.961,4.119,8.266,8.613,8.266c4.659,0,7.102-2.799,7.102-8.266V3.2h29.184v101.499    c0,14.307-1.856,20.506-9.11,27.762c-5.228,5.229-14.871,9.271-27.047,9.271c-9.837,0-19.25-3.256-25.253-9.27    c-5.263-5.273-8.154-10.689-12.672-27.764L44.9,37.033c-1.577-5.961-4.119-8.265-8.613-8.265c-4.66,0-7.103,2.798-7.103,8.265    v101.5H0v-101.5C0,22.727,1.857,16.527,9.111,9.271C14.337,4.044,23.981,0,36.158,0c9.837,0,19.25,3.257,25.253,9.27    c5.263,5.273,8.154,10.689,12.672,27.764L91.991,104.699z"
            fill="#e30613"
          />
          <path
            d="M478.038,138.533L444.334,33.096c-0.372-1.164-0.723-2.152-1.263-2.811    c-0.926-1.127-2.207-1.719-3.931-1.719c-1.723,0-3.004,0.592-3.931,1.719c-0.539,0.658-0.891,1.646-1.262,2.811l-33.703,105.437    h-30.167l36.815-115.177c1.918-6,4.66-11.094,8.139-14.488C421.002,3.047,428.038,0,439.141,0s18.14,3.047,24.109,8.867    c3.479,3.395,6.221,8.488,8.14,14.488l36.814,115.177H478.038z"
            fill="#e30613"
          />
          <path
            d="M328.878,138.533c19.12,0,28.446-4.062,35.814-11.389c8.153-8.105,12.053-16.973,12.053-30.213    c0-11.699-4.283-22.535-10.804-29.019c-8.526-8.479-19.116-11.151-36.384-11.151L305.37,56.76c-9.242,0-12.925-1.117-15.839-3.98    c-2.001-1.964-2.939-4.885-2.939-8.328c0-3.559,0.857-7.074,3.303-9.475c2.171-2.131,5.13-3.109,10.816-3.109h69.903V3.2H306.05    c-19.12,0-28.445,4.063-35.814,11.389c-8.152,8.105-12.053,16.972-12.053,30.212c0,11.701,4.283,22.536,10.804,29.019    c8.527,8.479,19.116,11.152,36.384,11.152l24.188,0.002c9.242,0,12.925,1.115,15.839,3.979c2.001,1.965,2.939,4.885,2.939,8.328    c0,3.559-0.857,7.074-3.302,9.475c-2.172,2.131-5.131,3.109-10.817,3.109h-72.094l-27.651-86.509    c-1.918-6-4.66-11.094-8.139-14.488C220.363,3.047,213.327,0,202.224,0s-18.14,3.047-24.108,8.867    c-3.48,3.395-6.221,8.488-8.139,14.488l-36.815,115.177h30.166l33.704-105.437c0.372-1.164,0.723-2.152,1.263-2.811    c0.926-1.127,2.208-1.719,3.931-1.719s3.004,0.592,3.931,1.719c0.54,0.658,0.891,1.646,1.262,2.811l33.704,105.437H328.878z"
            fill="#e30613"
          />
        </g>
      </svg>
      <h1
        className="NASA1975-heading NASA1975-heading--lg NASA1975-heading--light NASA1975-stem-logo-heading"
      >
        News
      </h1>
    </section>
    <div
      className="NASA1975-news-release-header"
    >
      <h2
        className="NASA1975-heading NASA1975-heading--xs NASA1975-heading--light NASA1975-agency-name NASA1975-agency-name--inline"
      >
        <span
          className="NASA1975-agency-name-N"
        >
          National
        </span>
         
        <span
          className="NASA1975-agency-name-Aa"
        >
          Aeronautics and
        </span>
         
        <span
          className="NASA1975-agency-name-SA"
        >
          <span
            className="NASA1975-agency-name-S"
          >
            Space
          </span>
           
          <span
            className="NASA1975-agency-name-A"
          >
            Administration
          </span>
        </span>
      </h2>
      
      <h2
        className="NASA1975-heading NASA1975-heading--xs NASA1975-heading--light NASA1975-agency-name-center-address"
      />
    </div>
    <ul
      className="NASA1975-article-author "
    >
      <li>
        Neil Armstrong
      </li>
      
      
    </ul>
    <h1
      className="NASA1975-heading NASA1975-heading--lg NASA1975-heading--medium "
    >
      Test title
    </h1>
  </header>
  <div
    className="NASA1975"
  >
    <p>
      Test article content
    </p>
  </div>
</article>
`;
